README.md

```diff
@@ -754,6 +754,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
 * Argument parsing
   * [clap-rs](https://github.com/clap-rs/clap) [[clap](https://crates.io/crates/clap)] — A simple to use, full featured command-line argument parser [<img src="https://api.travis-ci.org/clap-rs/clap.svg?branch=master">](https://travis-ci.org/clap-rs/clap)
+  * [owez/climake](https://github.com/Owez/climake) [[climake](https://crates.io/crates/climake)] - A simple, dependency-free cli framework
```

Comment:
Now there's a CI on that, can you add the build badge please? Sorry about all the stuff here, but this should be it to get it in. Please note that Github Actions badges should specifically reference their main branch (master in this case) to avoid potential cases where the build badge follows some other branch.

Reply:
No problem, my fault for forgetting

Reply:
The badge still needs the branch adding as per https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#example-using-the-branch-parameter